.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "annotation"]
+	path = annotation
+	url = https://github.com/rakrakon/annotation.git
```

Comment:
Probably not this URL.
Anyway, I highly recommend not using submodules, they usually create more problems than they solve. Why not just copy the relevant files (if it's more than one really) here, while giving proper credit? Or maybe doing something using JitPack?

Reply:
Proper credit is given in the README and in the package name. I wanted to have it in a separate repo because i intend to  program the @AutoLogOutput annotation aswell.

Reply:
And if you do have another repo it needs to be under the team's organization and not under your own account

Reply:
i am transferring it right now..

Reply:
Just because something has already been done, it doesn't mean it can't be reverted. It's fine to leave it there, but it's no excuse that it's been already done.